src/NServiceBus.Core/obsoletes.cs

```diff
@@ -686,7 +686,7 @@ public static string GetMessageHeader(this IBus bus, object msg, string key)
         [ObsoleteEx(
             RemoveInVersion = "7.0",
             TreatAsErrorFromVersion = "6.0",
-            Message = "Headers can be set using the ``.SetHeader` method on the context object passed into the behavior or mutator")]
+            Message = @"Use the overload of the Send, Publish or Reply method that accepts an options parameter. Call options.SetHeader(""MyHeader"",""MyValue"") instead.")]
```

Comment:
@indualagarsamy That line in the obsolete warning from the `ObsoleteEx` fody attribute, and it's referring to the implemenation stub left around for the warning. See https://github.com/Particular/NServiceBus/pull/3784/files#diff-287c0fdac06c80a87ce22cb7ee81ac17R692